lib/views/admin/newUser.js

```diff
@@ -16,15 +16,18 @@ module.exports = function (req, res) {
 }
 
 function form (req, res, locals) {
-  locals = extend({
-    config: config.get(),
-    section: 'admin',
-    adminSection: 'theme',
-    user: req.user,
-    form: locals.form || {}
-  }, locals)
-
-  res.send(pug.renderFile('templates/views/admin/newUser.jade', locals))
+  if (!req.accepts('text/html')) {
+    return res.status(200).header('content-type', 'application/json').send(JSON.stringify(locals))
```

Comment:
`locals` can contain the entire config, which contains values that we don't want to leak (like the synbiohub secret used to generate share links)

Reply:
Good catch.  Will just sent registerAlert.  BTW: must be admin to use these endpoints, but still see your point.